src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$gap};
+  width: #{$calendar-size};
+  padding: 10px;
+
+  @for $index from 28 through 31 {
+    &--month-length-#{$index} {
+      .calendar__day:nth-child(n + #{$index + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: #{$cell-size};
+    height: #{$cell-size};
+    box-sizing: border-box;
```

Comment:
Move this property to main.scss. It is a good tone to set it for the whole page and don't change without extra need

Reply:
I guess you meant only align items and box-sizing. If yes - I've changed it. Thank You.